### PR TITLE
Add bazooka.yml to build the project with bazooka

### PR DESCRIPTION
**Pull request by julienvey**
To build automatically the project using bazooka-ci http://docs.bazooka-ci.io/index.html

### CODE REVIEW

**Comment by ggiamarchi** [comment]
I see this `.bazooka.yml` is merely a copy of the existing `.travis.yml` with workarounds for Bazooka issues. As bazooka can handle `.travis.yml` descriptor, i would prefer to wait for bazooka bugfixes rather than merging this one.

File: .bazooka.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - cd source # https://github.com/bazooka-ci/bazooka/issues/251
+  - bundle _${BUNDLER_VERSION}_ install --without debug --retry=3
+
+script:
+  - cd source # https://github.com/bazooka-ci/bazooka/issues/251
+  - bundle _${BUNDLER_VERSION}_ exec rake
+
+env:
+  - VAGRANT_VERSION=v1.4.3
+  # - VAGRANT_VERSION=v1.5.4 See https://github.com/bazooka-ci/bazooka/issues/27#issuecomment-111121272
```
Comment (ggiamarchi) [inline .bazooka.yml]: Why there is a problem with this line specifically ? What's the difference with the others ?

**Comment by julienvey** [comment]
Bazooka will never fully support the `travis.yml` of this project. There are multiple reasons, first is the `travis_retry` command, second is the versions numbers of ruby that might (and currently are) be different

File: .bazooka.yml (hunk 1 above)
Comment (julienvey) [inline .bazooka.yml]: This line needs a different bundler version

see travis.yml
```
- VAGRANT_VERSION=v1.5.4 BUNDLER_VERSION=1.5.2
```

**Comment by ggiamarchi** [comment]
Closing because of a long inactivity and because Bazooka is not available as a SaaS.

**Pull request closed without merging**